.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
.vscode, .idea ?

Reply:
Sure! Added stuff for MacOS too, let me know if you think of more things.